conduit/schema/v1/grpc_client.go

```diff
@@ -0,0 +1,60 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import (
+	"context"
+	"fmt"
+
+	commonsschema "github.com/conduitio/conduit-commons/schema"
+	"github.com/conduitio/conduit-connector-protocol/conduit/schema"
+	"github.com/conduitio/conduit-connector-protocol/conduit/schema/v1/fromproto"
+	"github.com/conduitio/conduit-connector-protocol/conduit/schema/v1/toproto"
+	conduitv1 "github.com/conduitio/conduit-connector-protocol/proto/conduit/v1"
+	"google.golang.org/grpc"
+)
+
+type GRPCSchemaClient struct {
+	grpcClient conduitv1.SchemaServiceClient
+}
+
+func NewGRPCSchemaClient() (*GRPCSchemaClient, error) {
+	conn, err := grpc.NewClient("localhost:8085")
```

Comment:
Looking today more in depth the connector-protocol code I wondered if we should these ports configurable in case of collision.

Reply:
You're right about that. I've added that as part of https://github.com/ConduitIO/conduit/issues/1563.